app/views/users/sessions/email_sent.html.erb

```diff
@@ -18,11 +18,11 @@
           <label class="inline-block text-sm font-semibold tracking-wide text-gray-900" for="input_token">
             <%= t(".magic_link.verification_code") %>
           </label>
-          <%= f.text_field :input_token, maxlength: 6, minlength: 6, placeholder: t('.magic_link.code_placeholder'), required: true, class: "block w-full py-3 px-5 mt-2 text-lg tracking-[1.25em] text-gray-800 border border-gray-300 rounded appearance-none focus:outline-none focus:bg-white focus:border-primary-400", tabindex: 1, autocomplete: 'off', disabled: params[:input_tokens_deleted].present? %>
+          <%= f.text_field :input_token, maxlength: 6, minlength: 6, autofocus: true,  required: true, class: "block relative w-full text-center font-mono font-medium py-3 px-5 mt-2 tracking-[1.5rem] md:tracking-[2rem] text-2xl bg-gray-100 text-gray-800 border border-gray-300 rounded appearance-none focus:outline-none focus:bg-white focus:border-primary-400", tabindex: 1, autocomplete: 'off', disabled: params[:input_tokens_deleted].present? %>
```

Comment:
@vinutv The `bg-gray-100 focus:bg-white` on inputs is something I've removed everywhere in these forms. To my eyes, it made the input fields look disabled until it's focused on... I was thinking that the placeholder helped indicate to users the nature of the input as well, but that's been removed here. Was that intentional?

Reply:
@harigopal That's correct. When you mentioned that the input looks disabled for me with `bg-gray-100`, I'll change it back to `bg-white`.

The removal of the placeholder was intentional, and I should have explained why. Users might think the verification code has been autofilled, especially since the autofill feature is common for OTP fields on mobile phones. Using '123456' as a placeholder might lead to this confusion. Instead, we can use underscores ("_ _ _ _ _ _") as the placeholder :) to indicate that the field is empty and needs to be filled.

<img width="523" alt="Screenshot 2024-07-12 at 11 05 06 AM" src="https://github.com/user-attachments/assets/a210f633-78d6-42f1-aa6c-f3e15d6ae326">

<img width="506" alt="Screenshot 2024-07-12 at 11 11 58 AM" src="https://github.com/user-attachments/assets/c271d893-6487-4a88-95b2-53c7a0299aae">

Reply:
Commit - https://github.com/pupilfirst/pupilfirst/pull/1650/commits/5078e32ae7fc23075b135737c21ec3d46967c987

Reply:
@vinutv Underscore works!